convertor/lib/screens/price_reduction_calculator.dart

```diff
@@ -1,32 +1,140 @@
+import 'dart:ffi';
+
+import 'package:convertor/models/price_reduction.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
 
 class PriceReductionCalculator extends StatefulWidget {
   PriceReductionCalculator({Key key, this.title}) : super(key: key);
 
   final String title;
+  
+  
 
   @override
   _PriceReductionCalculatorState createState() =>
       _PriceReductionCalculatorState();
 }
 
+
+
+
 class _PriceReductionCalculatorState extends State<PriceReductionCalculator> {
+  
+  double prixSaisis; 
```

Comment:
```suggestion
  double inputPrice; 
```
Tout en anglais stp 👍